# rate at which the input messages are published (in Hz)
# double
target_publishing_rate: 20

# display timing information
# bool
display_timing: false
